sage-ios/SAGE/SAGE/NotificationConstants.swift

```diff
@@ -16,5 +16,6 @@ struct NotificationConstants {
     static let editAnnouncementKey: String = notificationBaseName + "editAnnouncement"
     static let addCheckinKey: String = notificationBaseName + "addCheckin"
     static let editProfileKey: String = notificationBaseName + "editProfile"
-    
+    static let addVerifiedCheckinKey: String = notificationBaseName + "addVerifiedCheckin"
+    static let addUnverifiedCheckinKey: String = notificationBaseName + "addUnverifiedCheckin"
```

Comment:
We be using these two keys instead of the addCheckinKey right -- so we should be able to get rid of that key and make it a case of one of these instead